SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -133,6 +133,7 @@ void USpatialReceiver::OnAddComponent(Worker_AddComponentOp& Op)
 	case SpatialConstants::SINGLETON_COMPONENT_ID:
 	case SpatialConstants::UNREAL_METADATA_COMPONENT_ID:
 	case SpatialConstants::INTEREST_COMPONENT_ID:
+	case SpatialConstants::HEARTBEAT_COMPONENT_ID:
```

Comment:
I feel like this switch statement is getting unwieldy, might want to make it a bit nicer in the future.

Reply:
This seems okay - but HandleActorAuthority() is becoming unwieldly.